### DISCUSSION TITLE
Can you help me with that?

### DISCUSSION THREAD

**Discussion by BoXjumP2002** [GriefPrevention/GriefPrevention#1318] [Q&A]
### What happened:
_my friend was muted for some reason_

### What was expected:
_That my friends are no longer muted_

### Steps to reproduce:
__I think he could have talked in the chat before moving like I was looking and sometimes for that. but im sure of nothing__
1. find the potential problem
2. confirm the problem.
3. fix the problem

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
Paste of `/version`: 
Paste of `/version GriefPrevention`: 16.17.1-2b69191

### GriefPrevention config.yml

GriefPrevention:
  SeaLevelOverrides:
    world: 256
    world_nether: 256
    world_the_end: 256
  Claims:
    Mode:
      world: Survival
      world_the_end: Survival
      world_nether: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: false
    LockWoodenDoors: false
    LockTrapDoors: false
    LockFenceGates: false
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 0
    Claim Blocks Accrued Per Hour:
      Default: 1
    Max Accrued Claim Blocks:
      Default: 1000000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: true
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: true
    FireDamagesInClaims: true
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: true
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: true
      world_nether: true
      world_the_end: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 60
    AllowCombatItemDrop: true
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: false
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: true
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: true
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 1000.0
    ClaimBlocksSellValue: 850.25
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  PistonMovement: EVERYWHERE
  PistonExplosionSound: false
  FireSpreads: true
  FireDestroys: true
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: true
  SilverfishBreakBlocks: true
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: true
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false

```yml

### Plugin list (if applicable): GriefPrevention, EssentialX, EssentialXGeo, EssentialXSpawn, Vault, ChestShop, GuiShop, Jobs, GroupManager, LimitePillagers, ( JUST FOR FUN -> Morph, Libdisguises, ProtocolLib )






Files just plugin and config
[All Plugin+  Config.zip](https://github.com/TechFortress/GriefPrevention/files/6141576/All.Plugin%2B.Config.zip)

**Comment by Jikoo** [comment]
Talking before moving without having ever picked up a log does not result in a mute, it only mutes the one message. Your friend probably said one of the words in your banned words list. Try `/softmute <your friend's name>` to see if they're soft muted.

**Comment by RoboMWM** [comment]
did this get converted from an issue or something? Anyways idk what's going on here but jikoo's suggestion is probably the one to follow

**Reply by Jikoo** [reply]
I don't believe so, I certainly didn't at least. I assumed it was an issue until I saw how my comment was formatted tbh
